Cargo.lock

```diff
@@ -46,9 +46,9 @@ dependencies = [
 
 [[package]]
 name = "anyhow"
-version = "1.0.48"
+version = "1.0.50"
```

Comment:
Not sure why it shows these as changes, in dev it's already `version = "1.0.50"`